coalib/testing/LocalBearTestHelper.py

```diff
@@ -26,20 +26,45 @@ def execute_bear(bear, *args, **kwargs):
     return list(bear_output_generator)
 
 
+def get_local_bear_run_settings_dict(local_bear, settings_dict):
```

Comment:
Right, I found a correct method in `FunctionMetadata` called `filter_parameters`